assembl/views/__init__.py

```diff
@@ -76,8 +84,10 @@ def find_theme(theme_name, frontend_version=1):
     @returns the theme path fragment relative to the theme base_path, or
     None if not found
     """
-    theme_base_path = get_theme_base_path(frontend_version)
+    current_theme = THEMES.get(theme_name, None)
+    if current_theme: return current_theme
```

Comment:
I don't like one liner like this, please add a new line and indent.

Reply:
:+1: